_opensearch/remote.md

```diff
@@ -97,21 +97,25 @@ curl -X PUT "https://localhost:9200/my-index?pretty" -ku admin:admin -H 'Content
   "settings": {
     "index": {
       "number_of_shards": 1,
-      "number_of_replicas": 0,
+      "number_of_replicas": 1,
       "replication": {
         "type": "SEGMENT"
       },
       "remote_store": {
         "enabled": true,
-        "repository": "my-repo-1"
+        "repository": "segment-repo",
+        "translog": {
+          "enabled": true,
+          "repository": "translog-repo"
+        }
       }
     }
   }
 }
 '
 ```
 
-All data that is added to the index will also be uploaded to the remote storage once it is committed.
+The segment & translog repository can be same or different. All data that is added to the index will also be continuously uploaded to the remote storage on account of refreshes / flushes / translog fsync (to disk) in the form of segment and translog files. Along with this, there are other metadata files uploaded.
```

Comment:
```suggestion
The segment repository and translog repository can be the same or different. As data is added to the index, it will also be continuously uploaded to the remote storage in the form of segment and translog files because of refreshes, flushes, and translog fsync to disk. Along with data, other metadata files will be uploaded.
```